src/services/webhook.service.ts

```diff
@@ -201,12 +202,13 @@ export class WebhookService {
   }
 
   private getEmbed(action: string, repositoryName: string, title: string, event: BaseEvent) {
+    const user = event.merged_by ? event.merged_by : event.user;
```

Comment:
```suggestion
    const user = event.merged_by || event.user;
```

I think this syntax is a thing right?

Reply:
Oh yeah woops, thanks!